internal/mode/static/nginx/config/http/config.go

```diff
@@ -86,3 +86,9 @@ type MapParameter struct {
 	Value  string
 	Result string
 }
+
+const (
```

Comment:
we don't need this constant anymore because it is not used in the location paths
more over, we don't need to validate HTTPRoute paths to not include this prefix -- this is no longer necessary, so we can safely remove the validation and the docs about it.

Reply:
yes make sense, updating

Reply:
updated